### ISSUE TITLE
No changelog?

### ISSUE THREAD

**Issue by dbackeus** [arunagw/omniauth-twitter#117]
A `CHANGELOG.md` file would be nice to understand what has happened when upgrading (currently upgrading from 0.0.16 to 1.4.0 and have no idea what's happened).

**Comment by arunagw** [comment]
@dbackeus Yes. Having a `CHANGELOG.md` will be very nice. I will try to add that soon. thanks!
